fastify.js

```diff
@@ -135,7 +136,17 @@ function fastify (options) {
   }
 
   // Instance Fastify components
-  const { logger, hasLogger } = createLogger(options)
+
+  // create a default logger and then overwrite if valid logger options provided
+  let logger = nullLogger
+  logger.child = () => logger
+  let hasLogger = false
```

Comment:
we may move it in the else condition so we would not need to run `require('abstract-logging')` if not needed too

Reply:
Cool. Done